templates/common/footer.tpl

```diff
@@ -14,7 +14,7 @@
 <div class="pkp_structure_footer" role="contentinfo">
 	<div class="pkp_brand_footer">
 		<a href="{$pkpLink}">
-			<img alt="{translate key="common.publicKnowledgeProject"}" src="{$baseUrl}/{$brandImage}">
+			<img alt="{translate key="common.publicKnowledgeProject"|default:'null'}" src="{$baseUrl}/{$brandImage}">
```

Comment:
You can leave this change out.

When you see a `{translate key="..."}` used, that comes directly from the locale strings. For that reason, you can be sure that the alt tag will never be empty.

Reply:
Ok! So I roll back my changes?

Reply:
Yep, so you can leave lib/pkp out of the changes altogether